### PR TITLE
Enabling VB in the CLI.

### PR DESCRIPTION
**Pull request by livarcocc**
Fixing the permissions for RunVbc.sh.

@dotnet/dotnet-cli @srivatsn

### CODE REVIEW

**Comment by livarcocc** [comment]
**Customer scenario**

Enabling VB in the CLI.

**Bugs this fixes**

[CLI #6615](https://github.com/dotnet/cli/issues/6615)

**Workarounds, if any**

N/A

**Risk**

Small. This change affects pretty much the new functionality only.

**Performance impact**

N/A

**Root cause analysis**

N/A

**How was the bug found?**

N/A

**Comment by livarcocc** [comment]
@dotnet-bot Test OSX10.12 x64 Release Build

**Comment by livarcocc** [comment]
@MattGertz for approval.

File: test/dotnet.Tests/GivenThatICareAboutVBApps.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+using Microsoft.DotNet.Tools.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.DotNet.Tests
+{
+    public class GivenThatICareAboutVBApps : TestBase
+    {
+        private TestAssetInstance _testInstance;
+
+        public GivenThatICareAboutVBApps()
```
Comment (wli3) [inline test/dotnet.Tests/GivenThatICareAboutVBApps.cs]: nit: GivenRestoredVBTestAppPorject

File: test/dotnet.Tests/GivenThatICareAboutVBApps.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+            new RestoreCommand()
+                .WithWorkingDirectory(_testInstance.Root)
+                .Execute()
+                .Should().Pass();
+        }
+
+        // Enable cross-plat once https://github.com/Microsoft/msbuild/issues/422 is fixed
+        [WindowsOnlyFact]
+        public void ICanBuildVBApps()
```
Comment (wli3) [inline test/dotnet.Tests/GivenThatICareAboutVBApps.cs]: nit: WhenInvokeBuildCommand ItCanBuildVBApps

**Comment by livarcocc** [comment]
https://github.com/Microsoft/msbuild/issues/422

**Comment by livarcocc** [comment]
Fixes CLI #6615

**Pull request merged**